src/scripts/eastern_kingdoms/swamp_of_sorrows/sunken_temple/instance_sunken_temple.cpp

```diff
@@ -618,6 +610,30 @@ InstanceData* GetInstance_instance_sunken_temple(Map* pMap)
     return new instance_sunken_temple(pMap);
 }
 
+/*######
+## go_atalai_light
+######*/
+
+struct go_atalai_lightAI : public GameObjectAI
+{
+    go_atalai_lightAI(GameObject* pGo) : GameObjectAI(pGo) {}
+
+    bool OnUse(Unit* pUser) override
+    {
+        if (ScriptedInstance* pInstance = (ScriptedInstance*)me->GetInstanceData())
+        {
+            pInstance->SetData64(me->GetEntry(), me->GetGUID());
```

Comment:
Isn't it possible to do this in db using `gameobject_scripts` table? You can assign script per guid of the spawn there.

Reply:
I have tried it with `Command: Set Data 64` and both Modes `Safe Own Guid` and `Save Raw Value` but it did not work.
This was my attempt with  `Save Raw Value`:
```
UPDATE `gameobject_template` SET `script_name`='' WHERE `entry` IN (148830, 148831, 148832, 148833, 148834, 148835);

DELETE FROM `gameobject_scripts` WHERE `id` IN (27898,27899,28111,28112,28113,28114);
INSERT INTO `gameobject_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) values
(27898, 0, 0, 38, 148830, 27898, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle'),
(27899, 0, 0, 38, 148831, 27899, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle'),
(28111, 0, 0, 38, 148832, 28111, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle'),
(28112, 0, 0, 38, 148833, 28112, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle'),
(28113, 0, 0, 38, 148834, 28113, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle'),
(28114, 0, 0, 38, 148835, 28114, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Atal\'ai Statue - process secret circle');
```
Could this problem be due to the db script not using the override method which handels the script and instead just using the base instance implementation? I have attached the debugger to the override method and nothing happend when I activated a stone in game.

Reply:
Converted to db. This did also fix the issue with the trap.